compareECLFiles.cmake

```diff
@@ -478,6 +478,24 @@ add_test_compareECLFiles(CASENAME norne
                          PREFIX compareECLInitFiles
                          DIR_PREFIX /init)
 
+add_test_compareECLFiles(CASENAME operate_work
```

Comment:
maybe we can move add_multiple_test_cases from regressionTests.cmake  and use it here?